etcdserver/cluster.go

```diff
@@ -393,7 +393,7 @@ func ValidateClusterAndAssignIDs(local *Cluster, existing *Cluster) error {
 	sort.Sort(SortableMemberSliceByPeerURLs(lms))
 
 	for i := range ems {
-		if !reflect.DeepEqual(ems[i].PeerURLs, lms[i].PeerURLs) {
+		if !netutil.URLStringsEqual(ems[i].PeerURLs, lms[i].PeerURLs) {
```

Comment:
i would like to merge this pr if you add a todo here in case we forget this is a quick fix.